enhancements/console/quick-starts.md

```diff
@@ -151,6 +155,30 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The
+Quick Start CRD `label` contains a `language` field to specify what
+language the Quick Start is in. CRDs without this label will be treated
+as English CRDs. OpenShift can display the correct
+Quick Starts for each language by comparing the value of the
+`language` label to the currently displayed language in OpenShift.
+(The display language is determined by `localStorage.getItem('bridge/language')`
+or the output of the comparison between the languages we support
+and `i18next.languages`.) This approach will not handle languages we don't
+offer support for; if Quick Starts in non-supported languages become a need,
+we will need to change this approach.
+
+The language label must be a two-letter
+[ISO 639-2 Language Code](https://www.loc.gov/standards/iso639-2/php/code_list.php))
+(i.e. "en" for English) or an ISO language code hyphenated with an alpha-2
+[ISO-3166 Country Code](https://www.iso.org/iso-3166-country-codes.html)
+(i.e. "en-us").
+
+The language is also appended to the name of the CRD so we can avoid showing
+duplicates of the same CRD content in different languages. The name and language are
+separated by a tilda ("~") so we can split them on a consistent delimiter.
```

Comment:
`~` is not typically allowed in k8s names. It's also used as our internal delimiter for group/version/kind in console, so I'd suggest not using it here. I'd say just `my-quick-start-{lang}`. It would be good to show an example YAML metadata with the label and name.

Reply:
Note that if we grow our localization, the language code can be 2 part separated by `-`. Perhaps our name to language code separator should be a `.` instead?
`<name>.fr-FR`